master/internal/workspace/postgres_workspace_intg_test.go

```diff
@@ -0,0 +1,136 @@
+//go:build integration
+// +build integration
+
+package workspace
+
+import (
+	"context"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/uptrace/bun"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/etc"
+	"github.com/determined-ai/determined/proto/pkg/workspacev1"
+)
+
+const (
+	cluster1 = "C1"
+	cluster2 = "C2"
```

Comment:
these dont help readability to me. was a linter complaining?

Reply:
I thought it was, but I removed this var and it worked